crates/bevy_winit/src/system.rs

```diff
@@ -428,3 +434,13 @@ pub(crate) fn changed_windows(
         cache.window = window.clone();
     }
 }
+
+#[derive(Default, Component)]
+pub struct WinitWindowPressedKeys(pub HashMap<winit::keyboard::PhysicalKey, PressedKey>);
+
+#[derive(PartialEq, Eq, Hash)]
+pub struct PressedKey {
+    pub logical_key: winit::keyboard::Key,
+    pub physical_key: winit::keyboard::PhysicalKey,
```

Comment:
Sorry, this might be a bit picky: What value does this field provide at the moment? As far as I can tell we would be fine without it since the `PhysicalKey` is already the key of the `HashMap`.

Reply:
True, this is mostly a leftover :ok_hand: And turns out I don't need the location either